.github/workflows/update-bidi-types.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: Generate TypeScript types for the main spec
         run: npm run bidi-types -- --cddl-file all.cddl
       - name: Generate TypeScript types for Permissions spec
-        run: npm run bidi-types -- --cddl-file permissions.cddl --ts-file src/protocol/generated/webdriver-bidi-permissions.ts --zod-file src/protocol-parser/generated/webdriver-bidi-permissions.ts
+        run: cddlconv -f zod permissions.cddl > src/protocol/generated/webdriver-bidi-permissions.ts
```

Comment:
this is the wrong destination and we need the FILE_HEADER. We also need TS types.